requirements/edx/pip-tools.txt

```diff
@@ -5,5 +5,5 @@
 #    make upgrade
 #
 click==7.0                # via pip-tools
-pip-tools==4.4.1
+pip-tools==4.5.0
```

Comment:
When I upgraded to pip-tools 4.5.0, I started getting extraneous diffs where it would add `# via -r requirements.txt` (or similar) to just about everything. Someone else want to check on this?

Reply:
That sounds like this: https://github.com/jazzband/pip-tools/pull/1058 .  That's actually a welcome change, right now it can be kind of tricky to figure out where some of the pinned dependencies came from (some of them are indirect dependencies from packages installed from VCS rather than direct dependencies.)  So it'll make a lot of changes, but I think the end result will be useful.  I'll go ahead and try it, and if it looks good add the commit to this branch.